frontend/apps/web/src/pages/all-tests/index.tsx

```diff
@@ -0,0 +1,285 @@
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsKinds.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <BarWrapper>
+          <Text type="text xl">All Tests Status</Text>
+          <ProgressBar
+            legend
+            onClick={() => navigate('/repository-details')}
+            data={allTestsStatus.map(({ name, percentage }) => ({
+              name,
+              percentage,
+            }))}
+          />
+        </BarWrapper>
+        <Heading>
+          <Text type="text xl">List of all tests ({allTests.length})</Text>
+        </Heading>
+        <Table
+          variant="default"
+          columns={{
+            suite: {
+              header: 'Suite',
+              width: 220,
+              Cell: ({ data }) => {
+                return (
+                  <>
+                    <Text type="text s">
+                      {data.id.split('/')[0]}
+                      <br />
+                    </Text>
+                    <Link
+                      to={data.id.split('/')[1]}
+                      appearance="system"
+                      icon="link"
+                      className="u-text--xsmall"
+                    >
+                      {data.id.split('/')[1]}
+                    </Link>
+                  </>
+                )
+              },
+            },
+            function: {
+              header: 'Function',
+              Cell: ({ data }) => {
+                return <Text type="text s">{data.functionName}</Text>
+              },
+            },
+            repository: {
+              header: 'Repository',
+
+              Cell: ({ data }) => {
+                return (
+                  <Link
+                    to={data.repository}
+                    appearance="system"
+                    className="u-text--xsmall"
+                  >
+                    {data.repository}
+                  </Link>
+                )
+              },
+            },
+            kinds: {
+              header: 'Kinds (tags)',
+
+              Cell: ({ data }) => {
+                return (
+                  <Link
+                    to={data.kind}
+                    appearance="system"
+                    className="u-text--xsmall"
+                  >
+                    {data.kind}
+                  </Link>
+                )
+              },
+            },
+            behaviors: {
+              header: 'Behaviors',
+
+              Cell: ({ data }) => {
+                return (
+                  <Text type="text s">
+                    {data.linkedBehaviors.length} behaviors
+                  </Text>
+                )
+              },
+            },
+            status: {
+              header: 'Status',
+              Cell: ({ data }) => {
+                if (data.status === 'pass') {
```

Comment:
Same goes for this as in Systems page for score.

Reply:
Done